packages/ua-devtools-evm-hardhat/src/tasks/oapp/wire/subtask.configure.ts

```diff
@@ -0,0 +1,56 @@
+import { SUBTASK_LZ_OAPP_WIRE_CONFIGURE } from '@/constants'
+import { OmniGraphBuilder, OmniTransaction } from '@layerzerolabs/devtools'
+import { createConnectedContractFactory, types } from '@layerzerolabs/devtools-evm-hardhat'
+import { createModuleLogger, printJson } from '@layerzerolabs/io-devtools'
+import { configureOApp, type OAppConfigurator, type OAppFactory, type OAppOmniGraph } from '@layerzerolabs/ua-devtools'
+import { createOAppFactory } from '@layerzerolabs/ua-devtools-evm'
+import { subtask } from 'hardhat/config'
+import { ActionType } from 'hardhat/types'
+
+export interface TaskArgs {
+    graph: OAppOmniGraph
+    configurator?: OAppConfigurator
+    oappFactory?: OAppFactory
+}
+
+const action: ActionType<TaskArgs> = async ({
+    graph,
+    configurator = configureOApp,
+    oappFactory = createOAppFactory(createConnectedContractFactory()),
+}: TaskArgs): Promise<OmniTransaction[]> => {
+    const logger = createModuleLogger(SUBTASK_LZ_OAPP_WIRE_CONFIGURE)
+
+    logger.verbose(`Running with graph:\n\n${printJson(graph)}`)
+
+    // As an additional step, even though this task is getting called
+    // from controlled and type-safe environments (for now),
+    // we pass thr graph through a builder
```

Comment:
nit:  `the`

Reply:
is fixed thank you they keep putting the letters too close on my keyboard